sdk/management/pipelines/fluentJavaMgmt.yml

```diff
@@ -1,14 +1,57 @@
 # External variables Settable at queue time:
 # "pomFile" : "pom.xml".   This is the project object model file for Maven.
 
+
 trigger:
-  - vnext
+  branches:
+    include:
+      - master
+  paths:
+    include:
```

Comment:
can we add some glob pattern here, otherwise in future each new service will need update this file?

Reply:
No, the devops trigger only support prefix. So every service needs to update it manually.

Reply:
Wildcard is not supported, so basically either include those that is needed, or include all under sdk and exclude those that is not needed...

Reply:
got it. thanks both. seems include is more safe now. after we stablized, may change it later